docs/home/whats-new-2-13.md

```diff
@@ -46,9 +51,12 @@ See [DCGM Metrics](https://docs.nvidia.com/datacenter/dcgm/latest/user-guide/fea
 <!-- RUN-8748/8958 RUN-9627/10483 WANDB-SWEEP & Run.ai integration / WANDB SWEEP Integration - phase 2 -->
 * Added sweep to Weights and Biases job submission. Sweep combines a strategy for exploring hyperparameter values with the code that evaluates them. The strategy can be as simple as trying every option or as complex as Bayesian Optimization and Hyperband (BOHB). To configure WANDB sweep, see [Sweep configuration](../admin/integration/weights-and-biases.md#sweep-configuration).
 
+<!-- Hagay: Support for Ray Jobs and Elastic Workloads. --> 
 <!-- RUN-9024/9027 Ray Support - schedule and support of Ray Jobs -->
 * Added support for Ray jobs. Ray is an open-source unified framework for scaling AI and Python applications. For more information, see [Integrate Run:ai with Ray](../admin/integration/ray.md#integrate-runai-with-ray).
 
+<!-- Hagay: what about DeepSpeed? OCP DevSpaces? did we mention it in any of the previous versions? -->
```

Comment:
@Hagay-RunAI DeepSpeed is in 2.12. OCP Dev Spaces, I did not see a ticket for. If tihs is needed for 2.13 please let me know. Reply with a ticket number and with the dev reponsible for the integration.